TestProjects/NotificationSamples/Assets/Scripts/ScriptableObjects/AndroidNotificationTemplate.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+#if PLATFORM_ANDROID
+using Unity.Notifications.Android;
+#endif
+
+[CreateAssetMenu(menuName = "Mobile Notifications/Android Notification Template")]
+public class AndroidNotificationTemplate : ScriptableObject
+{
+    #if PLATFORM_ANDROID
```

Comment:
No need for this define, let the code run in Editor too

Reply:
I added "|| UNITY_EDITOR" to these ifdefs in latest commit, because it doesn't compile if I remove them completely - e.g. when building on Android it tries to compile files that reference iOS namespace

Reply:
The reason is we only have Editor & iOS checked for Unity.Notification.iOS in the .asmdef. The same to Android.
So either we add #ifdef or we add different .asmdef for the testing script.